lib/msal-common/src/client/RefreshTokenClient.ts

```diff
@@ -25,18 +25,19 @@ import { UrlString } from "../url/UrlString";
 import { CcsCredentialType } from "../account/CcsCredential";
 import { buildClientInfoFromHomeAccountId } from "../account/ClientInfo";
 import { InteractionRequiredAuthError, InteractionRequiredAuthErrorMessage } from "../error/InteractionRequiredAuthError";
-import { IPerformanceClient } from "../telemetry/performance/IPerformanceClient"; 
 import { PerformanceEvents } from "../telemetry/performance/PerformanceEvent";
-
+import { IPerformanceClient } from "../telemetry/performance/IPerformanceClient";
 /**
  * OAuth2.0 refresh token client
  */
 export class RefreshTokenClient extends BaseClient {
     constructor(configuration: ClientConfiguration, performanceClient?: IPerformanceClient) {
-        super(configuration,performanceClient);
-    }
+        super(configuration, performanceClient);
```

Comment:
Addressed based on previous feedback, maintaining performanceClient as separate parameter to be passed to the RefreshTokenClient and SilentFlowClient